delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/indexing/CompositeIndexingCoordinator.scala

```diff
@@ -0,0 +1,79 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.indexing
+
+import akka.actor.typed.ActorSystem
+import ch.epfl.bluebrain.nexus.delta.kernel.RetryStrategy
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.UUIDF
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.BlazegraphViews
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.CompositeViews
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.config.CompositeViewsConfig
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model.CompositeView
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.MigrationState
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.views.indexing.IndexingStreamCoordinator
+import ch.epfl.bluebrain.nexus.delta.sdk.views.model.ViewIndex
+import ch.epfl.bluebrain.nexus.delta.sourcing.projections.ProjectionId.ViewProjectionId
+import com.typesafe.scalalogging.Logger
+import monix.bio.{IO, Task}
+import monix.execution.Scheduler
+
+object CompositeIndexingCoordinator {
+
+  type CompositeIndexingCoordinator = IndexingStreamCoordinator[CompositeView]
+
+  implicit private val logger: Logger = Logger[CompositeIndexingCoordinator]
+
+  private def fetchView(views: CompositeViews, config: CompositeViewsConfig) = (id: Iri, project: ProjectRef) =>
+    views
+      .fetch(id, project)
+      .map { res =>
+        Some(
+          ViewIndex(
+            res.value.project,
+            res.id,
+            res.value.uuid,
+            ViewProjectionId("none"),
```

Comment:
is this right?

Reply:
Yes. There is no projection for the whole view, but for each compositeView.projection